README.md

```diff
@@ -2,13 +2,13 @@ shavar-list-creation
 ====================
     usage: lists2safebrowsing.py
 
-A shavar.ini file must exist in the same directory that specifies the following parameters. See sample_shavar_list_creation.ini for examples.
+A `shavar_list_creation.ini` file must exist in the same directory that specifies the following parameters. (See `sample_shavar_list_creation.ini` for examples.)
 
-Commits to shavar-list-exceptions trigger rebuilds of shavar-list-creation like so:
+Commits to [shavar-list-exceptions](https://github.com/mozilla-services/shavar-list-exceptions) trigger rebuilds of [shavar-list-creation](https://github.com/mozilla-services/shavar-list-creation) like so:
```

Comment:
We should update this because there is another trigger now: commits to https://github.com/mozilla-services/shavar-prod-lists

@ckolos can confirm what the triggers are